src/assets/createScript.js

```diff
@@ -82,4 +83,29 @@ ${className}.prototype.update = function(dt) {
     `.trim();
 }
 
+function createEsmBoilerplate(className, scriptName) {
+    return `
+export class ${className} extends pc.ScriptType {
+
+    static name = '${scriptName}';
+
+    initialize() {
+
+    }
+
+    update() {
+
+    }
+
+    // swap method called for script hot-reloading
+    // inherit your script state here
+    // swap(old) { };
+}
+
+// to learn more about script anatomy, please read:
+// https://developer.playcanvas.com/en/user-manual/scripting/
```

Comment:
```suggestion
// learn more about script anatomy here:
// https://developer.playcanvas.com/user-manual/scripting/
```

Reply:
Maybe we also want this across both esm and classic scripts?

Reply:
Indeed.